planning/route_handler/src/route_handler.cpp

```diff
@@ -1271,11 +1271,26 @@ lanelet::ConstLanelets RouteHandler::getCheckTargetLanesFromPath(
   return check_lanelets;
 }
 
-lanelet::routing::RoutingGraphContainer RouteHandler::getOverallGraph() const
+bool RouteHandler::isMapMsgReady() const { return is_map_msg_ready_; }
+
+lanelet::routing::RoutingGraphPtr RouteHandler::getRoutingGraphPtr() const
+{
+  return routing_graph_ptr_;
+}
+
+lanelet::traffic_rules::TrafficRulesPtr RouteHandler::getTrafficRulesPtr() const
 {
-  return *overall_graphs_ptr_;
```

Comment:
replaced unsafe function to return pointer to function to return shared_ptr  itself